PeerioTechnologies/peerio-desktop
# fix: check chat.metaLoaded, fix empty DM titles

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8955/desktop-chatview-doesn-t-check-if-activechat-chathead-is-null-and-renders-and-crashes

#### Testing instructions  
* DM titles were empty. Receipient's name should be there again.
* Skylar got a blue screen of death due to a null `chatHead`. Wasn't able to repro but there is now a check so chats will not load if `metaLoaded` is false (so, can only load if `chatHead` exists)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
